source/Nuke.Common/Execution/Host.cs

```diff
@@ -47,21 +47,23 @@ protected internal void WriteLogo()
             Debug();
         }
 
-        protected internal virtual IDisposable WriteBlock(string text)
+        protected internal virtual IDisposable WriteBlock(string text, bool enableHeaderOutput = true)
```

Comment:
I am not so sure if adding this parameter to each and every Host implementation is so nice.